src/platform/Linux/DiagnosticDataProviderImpl.cpp

```diff
@@ -275,6 +289,8 @@ CHIP_ERROR DiagnosticDataProviderImpl::ResetWatermarks()
 
     // On Linux, the write operation is non-op since we always rely on the mallinfo system
     // function to get the current heap memory.
```

Comment:
This comment no longer matches reality @Thirsrin

Reply:
Thanks for addressing it @bzbarsky-apple, created a PR for this https://github.com/project-chip/connectedhomeip/pull/33309

Reply:
Thank you!